yeat/cli.py

```diff
@@ -138,6 +140,13 @@ def get_parser(exit_on_error=True):
         default=150,
         help="target an average depth of coverage Cx when auto-downsampling; by default, C=150",
     )
+    parser.add_argument(
+        "--seed",
+        type=int,
+        metavar="S",
+        default=None,
+        help="override the randomly chosen seed S when downsampling; by default, S=None",
+    )
```

Comment:
The metavariable is useful when describing default numerical or string values, but in this case `S=None` doesn't convey the corresponding behavior. I'd suggest describing the default behavior instead. For your consideration:

> seed for the random number generator used for downsampling; by default the seed is chosen randomly